aws/resource_aws_vpc_endpoint_service.go

```diff
@@ -115,7 +119,39 @@ func resourceAwsVpcEndpointServiceRead(d *schema.ResourceData, meta interface{})
 		return nil
 	}
 
-	return vpcEndpointServiceAttributes(d, svcCfg.(*ec2.ServiceConfiguration), conn)
```

Comment:
Replace call to `vpcEndpointServiceAttributes()` with explicit setting of attributes (including new `manages_vpc_endpoints` attribute).